vsts_ci/.vsts-ci.yml

```diff
@@ -42,6 +42,6 @@ jobs:
         Python37:
           python.version: "3.7"
           TOXENV: "py37"
-      maxParallel: 3
+      maxParallel: 1
```

Comment:
why reduce the maxParallel?

Reply:
When push or deploy in same time, the job will fail.